broadcast/internal/arc/arc_submit_tx.go

```diff
@@ -203,3 +227,42 @@ func validateSubmitTxResponse(model *broadcast.SubmittedTx) error {
 
 	return nil
 }
+
+func convertToEfTransaction(tx *SubmitTxRequest) error {
+	junglebusClient, err := junglebus.New(
+		junglebus.WithHTTP("https://junglebus.gorillapool.io"),
+	)
+	if err != nil {
+		return err
+	}
+	transaction, err := bt.NewTxFromString(tx.RawTx)
+	if err != nil {
+		return err
+	}
+	for _, input := range transaction.Inputs {
+		txid := input.PreviousTxIDStr()
+		var tx *models.Transaction
+		if tx, err = junglebusClient.GetTransaction(context.Background(), txid); err != nil {
+			return err
+		} else {
+			actualTx, err := bt.NewTxFromBytes(tx.Transaction)
+			if err != nil {
+				return err
+			}
+			o := actualTx.Outputs[input.PreviousTxOutIndex]
+			input.PreviousTxScript = o.LockingScript
+			input.PreviousTxSatoshis = o.Satoshis
+		}
```

Comment:
```suggestion
		
```

Reply:
I think we can remove else here, the Go convention is more like:
if err != nil {
 // Do something/return something
}

rest of the code

Reply:
Sure thing, changed.